src/Symfony/Component/Mime/Tests/AbstractMimeTypeGuesserTestCase.php

```diff
@@ -116,7 +116,8 @@ public function testGuessWithNonReadablePath()
         touch($path);
         @chmod($path, 0333);
 
-        if (str_ends_with(sprintf('%o', fileperms($path)), '0333')) {
+        $perms = @fileperms($path);
+        if (false !== $perms && str_ends_with(sprintf('%o', $perms), '0333')) {
```

Comment:
just adding the `@` would provide the same behavior - but this is a test case so I'd rather see the warning, unless it shows up too often?